### PR TITLE
Version 1.7.0-Beta

### PR DESCRIPTION
**Pull request by qwwdfsad**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+
+* New `Channel` implementation with significant performance improvements across the API (#3621).
+* New `select` operator implementation: faster, more lightweight, and more robust (#3020).
+* `Mutex` and `Semaphore` now share the same underlying data structure (#3020).
+* `Dispatchers.IO` is added to K/N (#3205)
+  * `newFixedThreadPool` and `Dispatchers.Default` implementations on K/N were wholly rewritten to support graceful growth under load (#3595).
+* `kotlinx-coroutines-test` rework:
+  - Add the `timeout` parameter to `runTest` for the whole-test timeout, 10 seconds by default (#3270). This replaces the configuration of quiescence timeouts, which is now deprecated (#3603).
+  - The `withTimeout` exception messages indicate if the timeout used the virtual time (#3588).
+  - `TestCoroutineScheduler`, `runTest` and `TestScope` API are promoted to stable (#3622).
```
Comment (dkhalanskyjb) [inline CHANGES.md:15]:
```suggestion
  - `TestCoroutineScheduler`, `runTest`, and `TestScope` API are promoted to stable (#3622).
  - `runTest` now also fails if there were uncaught exceptions in coroutines not inherited from the test coroutine (#1205).
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+### Bug fixes and improvements
+
+* Kotlin version is updated to 1.8.10.
+* JPMS is supported (#2237). Thanks @lion7!
+* `BroadcastChannel` and all the corresponding API are deprecated (#2680).
+* Added all supported K/N targets (#3601, #812, #855).
+* K/N `Dispatchers.Default` is backed by the number of threads equal to the number of available cores (#3366).
+* Fixed an issue where some coroutines' internal exceptions were not properly serializable (#3328).
+* Introduced `Job.parent` API (#3201).
+* Fixed a bug when `TestScheduler` leaked cancelled jobs (#3398).
```
Comment (dkhalanskyjb) [inline CHANGES.md:34]:
```suggestion
* `TestScope.timeSource` now provides comparable time marks (#3617). Thanks @hfhbd!
```

File: CHANGES.md (hunk 1 above)
Comment (dkhalanskyjb) [reply CHANGES.md]: I don't see the suggested new line anywhere.

Comment (qwwdfsad) [reply CHANGES.md]: Oh, I see -- I've applied the comma, but missed the next line

**Review by dkhalanskyjb — APPROVED** [review]

**Pull request merged**